pavelib/assets.py

```diff
@@ -292,6 +301,27 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
```

Comment:
I wasn't aware of a difference in update_assets between sandboxes and production. What kind of configuration change are you thinking of? My naive thinking was that this would get called the same way in all environments. It is very similar to how a lot of the other static assets are handled, e.g. Sass compilation, XModule bundling etc. Do they require separate prod configuration?